src/components/FileViewer/FileContextMenu.tsx

```diff
@@ -90,9 +96,20 @@ const FileContextMenu = forwardRef(
             // subLabel={ isMac ? "⌘+X" : "CTRL+X"}
             onClick={() => {
               cutFile();
+              setShowContextMenu(false);
             }}
             disabled={file === null}
           />
+          <ContextMenuButton
+            className="w-full whitespace-nowrap rounded-none bg-gray-900 px-2.5 text-small font-medium"
+            label="Paste"
+            // subLabel={ isMac ? "⌘+C" : "CTRL+C"}
+            onClick={() => {
+              pasteFiles((file === null  || (file as ClientFile).file_type !== 'Directory') ? currentPath : (file as ClientFile).path);
```

Comment:
It would be nice to make this more readable and do something like 
`file?.file_type === 'Directory' ? file.path : currentPath`
if TS compiler is smart enough to allow this that is